examples/ssoadmin/customermanagedpolicyattachment.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: ssoadmin.aws.upbound.io/v1beta1
+kind: CustomerManagedPolicyAttachment
+metadata:
+  annotations:
+    meta.upbound.io/example-id: ssoadmin/v1beta1/customermanagedpolicyattachment
+  labels:
+    testing.upbound.io/example-name: ssoadmin-customer-managed-policy-attachment
+  name: ssoadmin-customer-managed-policy-attachment
+spec:
+  deletionPolicy: Delete
+  forProvider:
+    customerManagedPolicyReference:
+    - policyNameSelector:
+        matchLabels:
+          testing.upbound.io/example-name: ssoadmin-customer-managed-policy-attachment
+      path: "/"
+    instanceArnSelector:
+      matchLabels:
+        testing.upbound.io/example-name: ssoadmin-customer-managed-policy-attachment
+    permissionSetArnSelector:
+      matchLabels:
+        testing.upbound.io/example-name: ssoadmin-customer-managed-policy-attachment
+    region: us-east-1
+
+---
+
+apiVersion: iam.aws.upbound.io/v1beta1
+kind: Policy
+metadata:
+  annotations:
+    meta.upbound.io/example-id: ssoadmin/v1beta1/customermanagedpolicyattachment
```

Comment:
```suggestion
    meta.upbound.io/example-id: ssoadmin/v1beta1/customermanagedpolicyattachment
    upjet.upbound.io/manual-intervention: "This resource is skipping because the parent resource was skipped."
```